### DISCUSSION TITLE
RustDesk Android - Send Intent to set ID Server/Key?

### DISCUSSION THREAD

**Discussion by plia7** [rustdesk/rustdesk#7840] [General]
Hello,

Is there a send intent for RustDesk Android to set ID Server/Key?
If not, can it be added?

Thanks.

**Comment by rustdesk** [comment]
https://twitter.com/rustdesk/status/1782250852574769663

**Reply by plia7** [reply]
> https://twitter.com/rustdesk/status/1782250852574769663

Sorry not sure what does this mean?
